src/client/components/AreaSelector/CountryList/CountryListRow/CountryListRow.tsx

```diff
@@ -49,16 +50,16 @@ const CountryListRow: React.FC<Props> = (props: Props) => {
 
   return (
     <div
+      aria-hidden="true"
       className={classNames('country-selection-list__row', role, { expanded, selected })}
       onClick={(e) => {
         e.preventDefault()
         onElementSelect(countryIso)
       }}
-      aria-hidden="true"
     >
       <div ref={countryNameRef}>{i18n.t<string>(Areas.getTranslationKey(countryIso))}</div>
 
-      {hasRole && (
+      {hasRole && !Objects.isEmpty(country) && (
```

Comment:
After looking into how we fetch the areas, we check [if atlantis is allowed](https://github.com/openforis/fra-platform/blob/development/src/server/repository/assessmentCycle/country/getMany.ts#L17-L20) for the assessment/cycle. Since atlantis is not allowed for FRA 2020 and the user has a role in it,  the `useCountry` hooks returns undefined and later the app crashes when the properties are accessed. 
Is this actually a data issue? that we have user roles in atlantis FRA 2020 when atlantis is not allowed for it.

Reply:
Added it :+1: 
And I'm keeping this `!Objects.isEmpty(country)` here because if you switch cycles fast enough or the network is slow, the same issue will be present while the new countries info is fetched.